src/tasks/leveling.ts

```diff
@@ -392,46 +450,108 @@ export const LevelingQuest: Quest = {
       name: "Wish for XP% buff",
       // TODO: Make this completed if we've already wished twice with the paw (requires mafia tracking)
       completed: () =>
-        have($effect`Different Way of Seeing Things`) ||
+        have(xpWishEffect) ||
         !have($item`cursed monkey's paw`) ||
-        forbiddenEffects.includes($effect`Different Way of Seeing Things`) ||
+        forbiddenEffects.includes(xpWishEffect) ||
         get("instant_saveMonkeysPaw", false) ||
-        myBasestat($stat`Mysticality`) >= targetBaseMyst - targetBaseMystGap ||
+        myBasestat(myPrimestat()) >= targetBaseMyst - targetBaseMystGap ||
```

Comment:
We should rename `targetBaseMyst` to `targetBasestat`, and also properly handle the transition of the relevant preference